src/devices/GoPiGo3/Movements/Motor.cs

```diff
@@ -0,0 +1,203 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.GoPiGo3.Models;
+using System;
+using System.Threading;
+
+namespace Iot.Device.GoPiGo3.Movements
+{
+    /// <summary>
+    /// Polarity of the motor
+    /// </summary>
+    public enum Polarity
+    {
+#pragma warning disable
```

Comment:
What warning are we disabling here and why?

Reply:
looks like this is not needed, you are right. .NET Core does not raise any warning for negative enum